sample/src/main/java/com/fueled/annotics/sample/TrackedClass.java

```diff
@@ -31,7 +31,7 @@ public void updateCardInformation(String cardNumber, String expDate, String cvv)
      * specifying the {@link EventValue#hidden()} as {@code true}.
      */
     @TrackEvent("EVENT_LOGIN")
-    public void login(String email, @EventValue(hidden = true) String password) {
+    public void login(String email, @IgnoreParam String password) {
```

Comment:
@arun-fueled let's hold on updating the samples for now. We can update it once we've published a new version.